packages/account/src/providers/provider.ts

```diff
@@ -775,6 +774,46 @@ export default class Provider {
     };
   }
 
+  /**
+   * Estimates the transaction gas and fee based on the provided transaction request.
+   * @param transactionRequest - The transaction request object.
+   * @param optimizeGas - Optional. Specifies whether to optimize the gas. Default is false.
+   * @returns An object containing the estimated minimum gas, minimum fee, maximum gas, and maximum fee.
+   */
+  estimateTxGasAndFee(params: { transactionRequest: TransactionRequest; optimizeGas?: boolean }) {
+    const { transactionRequest } = params;
```

Comment:
The parameter `optimizeGas` it not in use.

```suggestion
  /**
   * Estimates the transaction gas and fee based on the provided transaction request.
   * @param transactionRequest - The transaction request object.
   * @returns An object containing the estimated minimum gas, minimum fee, maximum gas, and maximum fee.
   */
  estimateTxGasAndFee(transactionRequest: TransactionRequest) {
```

Reply:
@Torres-ssf I see you kept the `params` object instead, and that's fine, so I'm resolving this.